Lib/test/support/__init__.py

```diff
@@ -43,6 +43,7 @@
     "anticipate_failure", "load_package_tests", "detect_api_mismatch",
     "check__all__", "skip_if_buggy_ucrt_strfptime",
     "check_disallow_instantiation",
+    "requires_subprocess", "requires_fork",
```

Comment:
Your test changes for forks are already merged, right?

Reply:
Yes, I split the PR into two PRs and forgot to clean up this line.